rules/test/content/config.yaml

```diff
@@ -0,0 +1,599 @@
+# Copyright 2020 Red Hat, Inc
```

Comment:
this whole file is not needed, it is already copied by the update_rules_content.sh script

Reply:
True